tutorial/extensions_dev.md

```diff
@@ -66,3 +66,52 @@ throughout.  **Phosphor messages** are a *many-to-one* interaction that allows
 information like resize events to flow through the widget hierarchy in
 the application.  **Phosphor signals** are a *one-to-many* interaction that allow
 listeners to react to changes in an observed object.
+
+
+## Extension Authoring
+An Extension is a valid npm package that meets the following criteria:
```

Comment:
Perhaps gloss or link to a definition of npm for people more accustomed to the Python ecosystem?